### DISCUSSION TITLE
Inverter IP Adress changed

### DISCUSSION THREAD

**Discussion by Amoskills** [weltenwort/home-assistant-rct-power-integration#287] [Q&A] [answered]
Hey, thanks for this awesome Integration.
But my Inverters IP changed. How do i change it without reinstalling the integration?

Thanks!

**Comment by weltenwort** [comment, marked as answer]
Hi @Amoskills :wave:

unfortunately I couldn't find an official way in HA to offer a reconfiguration. But the integration's config entry uses the inverter's serial number as the unique id, so you could try to add the integration a second time (via the "+" button on the "Integrations" screen) using the new IP address. If I understand the HA config flow correctly that should cause it to replace the existing config entry.

Let me know if that works.

**Comment by Amoskills** [comment]
Oh that was easy, it works. Good to know for the future. :)
Thanks and have a nice evening.
